BridgeEmulator/web-ui/src/index.js

```diff
@@ -19,13 +23,16 @@ const enhance = compose(
   // entity may be a light or a room, they have an extra `id` property
   // the second argument is the new value selected by the user
   withProps({
-    onColorTemperatureChange: (entity, temp) => console.log(entity.id, temp),
-    onColorChange: (entity, color) => console.log(entity.id, color),
-    onBrightnessChange: (entity, bri) => console.log(entity.id, bri),
-    onStateChange: (entity, state) =>
-      console.log(entity.id, entity.type, state),
+    onColorTemperatureChange: throttle((entity, temp) =>
```

Comment:
Now the API calls will not run more than once per second

Reply:
Basically i need to add `throttle((entity, value))` instead of (entity, value), but i receive errors on data compilation:
```
Failed to compile.

./src/index.js
Syntax error: Unexpected token, expected , (36:55)

  34 |   withProps({
  35 |     //onColorTemperatureChange: (entity, temp) => httpPutRequest(`/api/${API_KEY}/lights/${entity.id}/state`,{"ct": temp}),
> 36 |     onColorTemperatureChange: throttle((entity, temp)) => httpPutRequest(`/api/${API_KEY}/lights/${entity.id}/state`,{"ct": temp}),
     |                                                        ^
  37 |     onColorChange: (entity, color) => console.log(entity.id, color),
  38 |     onBrightnessChange: (entity, bri) => httpPutRequest(`/api/${API_KEY}/lights/${entity.id}/state`,{"bri": bri}),
  39 |     onStateChange: (entity, state) =>


npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! hue-emulator-ui@1.0.0 build: `react-scripts build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the hue-emulator-ui@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/marius/.npm/_logs/2018-05-17T08_26_55_286Z-debug.log
```

Reply:
i believe i need to import the library first:
import { Throttle } from 'react-throttle';

Reply:
Your code is compiling, so is my having an error in local changes.

Reply:
You forgot to close the parenthesis.

Reply:
I saw, not i will commit here also my changes.